RallyHereIntegration/Source/RallyHereEditor/Public/RallyHereEditorLoginWidget.h

```diff
@@ -0,0 +1,27 @@
+// Copyright Epic Games, Inc. All Rights Reserved.
+#pragma once
+
+#include "Widgets/Docking/SDockTab.h"
```

Comment:
Likely should be including SCompoundWidget instead of dock tab?

Reply:
I think we need both due to construct

void Construct(const FArguments& InArgs, const TSharedRef<SDockTab>& ConstructUnderMajorTab, const TSharedPtr<SWindow>& ConstructUnderWindow);